com.chartboost.mediation/Runtime/Platforms/ChartboostMediationExternal.cs

```diff
@@ -44,11 +45,13 @@ public virtual void Init()
 
         /// Initialize the Chartboost Mediation plugin with a specific appId
         /// Either one of the init() methods must be called before using any other Chartboost Mediation feature
-        public virtual void InitWithAppIdAndSignature(string appId, string appSignature)
-        {
-            Logger.Log(LogTag, $"InitWithAppIdAndSignature {appId}, {appSignature} and version {Application.unityVersion}");
-        }
-        
+        [Obsolete("InitWithAppIdAndSignature has been deprecated, please use InitWithOptions instead")]
+        public virtual void InitWithAppIdAndSignature(string appId, string appSignature) 
+            => Logger.Log(LogTag, $"InitWithAppIdAndSignature {appId}, {appSignature} and version {Application.unityVersion}");
+
+        public virtual void InitWithOptions(string appId, string appSignature, string[] initializationOptions = null) 
+            => Logger.Log(LogTag, $"InitWithAppIdAndSignature {appId}, {appSignature} and version {JsonConvert.SerializeObject(initializationOptions)}");
```

Comment:
Should this be something like:
```suggestion
            => Logger.Log(LogTag, $"InitWithOptions {appId}, {appSignature}, options {JsonConvert.SerializeObject(initializationOptions)} and version {Application.unityVersion}");
```